### ISSUE TITLE
Cannot be reproduced

### ISSUE THREAD

**Issue by deahhh** [deepinsight/insightface#2051]
It seems that this open source implementation mainly uses the `mxnet` framework. But why do I call `train.py` from the `arcface_mxnet` subdirectory with the author's pre-made dataset without pre-trained model and still fail to reproduce the results shown. The current training is 9 rounds, acc=0.646, and the correct rates on the validation set are 0.9935, 0.91129, and 0.93167 respectively.
Other than that, I noticed that the data preprocessing process did not subtract the mean and did not divide by the standard deviation.
My command is:
`CUDA_VISIBLE_DEVICES='0' nohup python -u train.py --network r50 --loss cosface --dataset emore >log &`

Any buddy can give me a hand?
Thanks in advance!

**Comment by nttstar** [comment]
the total batch size should be at least 512.

**Comment by deahhh** [comment]
Thank you very much!
Can you advise on the differences between the `mxnet` and `pytorch` versions in terms of pre-processing input normalisation? I am not familiar with `mnxet`. That's about dividing 255 and subtracting [0.5,0.5,0.5].
